vql/linux/audit.go

```diff
@@ -60,6 +105,9 @@ func (self AuditPlugin) Call(
 			return
 		}
 
+                arg := _AuditPluginArgs{}
+                err = arg_parser.ExtractArgsWithContext(ctx, scope, args, &arg)
+
 		client, err := libaudit.NewMulticastAuditClient(nil)
```

Comment:
This client is documented as not being for use in command and control. We'll need a separate client for that.